content/post/language_migrations.md

```diff
@@ -0,0 +1,512 @@
+---
+author: waren
+date: 2017-06-13
+title: "Language & History on GitHub"
+draft: false
+image: /post/language_migrations/background.png
+description: "This post is inspired by : the eigenvector of why we moved from language X to language Y, by Erik Bernhardsson. Based on GitHub repositories, we build our own transition matrix thanks to flow optimization."
+categories: ["science", "technical"]
+---
+<style>
+p.caption {
+  margin-top: -16px;
+  font-style: italic;
+}
+.grid2x {
+  display: flex;
+  width: 1000px;
+  overflow: visible;
+  margin-left: -200px;
+}
+.grid2x pre {
+  text-align: center;
+}
+@media (max-width: 1000px) {
+  .grid2x {
+    width: 100%;
+    margin-left: 0;
+  }
+  .grid2x > div {
+    width: 50%;
+  }
+  .grid2x pre {
+    font-size: 0.75em;
+  }
+}
+</style>
```

Comment:
@warenlg You cannot stop using custom styles because this code defines the adaptive grid. @Serabe Is it possible to move this into some common blog CSS file? p.caption is used in 10 articles, and the grid is used in 2 articles.